graceoncampus/goc-web-v3
# Update radio button styling + fix title of "Turnarounds" to "Pickup Location

Testing Steps
------------------------------------------------------------------------
1. yarn && yarn build && yarn start
2. navigate to localhost:3000
3. ensure that the proposed changes match up to the mock

Resources
------------------------------------------------------------------------
- css selectors: https://www.w3schools.com/cssref/css_selectors.asp
- react-bootstrap: https://react-bootstrap.github.io/
